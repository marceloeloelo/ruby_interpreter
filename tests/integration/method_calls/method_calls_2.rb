puts ("Hello".length())

puts ("H".length() * 2)

puts ("Hello".length() + "Goodbye".length())

puts ("".length())
