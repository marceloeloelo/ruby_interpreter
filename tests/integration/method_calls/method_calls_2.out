5
2
12
0
false
false
false
true
true
false
true
false
true
false
true
false
true
false
